linodecli/__init__.py

```diff
@@ -21,17 +21,14 @@
     remove_plugin,
 )
 from .cli import CLI
-from .completion import bake_completions, get_completions
+from .completion import get_completions
 from .configuration import ENV_TOKEN_NAME
 from .help_pages import print_help_action, print_help_default
 from .helpers import handle_url_overrides
 from .output import OutputMode
+from .version import __version__
 
-# this might not be installed at the time of building
-try:
-    VERSION = version("linode-cli")
-except:
-    VERSION = "building"
+VERSION = __version__
```

Comment:
This should be much more reliable than grabbing the version from the package itself.